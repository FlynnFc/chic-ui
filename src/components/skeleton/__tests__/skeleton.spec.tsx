import Skeleton from '..';
import { render } from '../../../test/utils';

describe('Skeleton Component', () => {
  it('Renders Correctly', () => {
    const SkeletonGroup = render(
      <>
        <Skeleton />
        <Skeleton width={80} height={60} />
        <Skeleton borderRadius="50%" />
      </>
    );
    expect(SkeletonGroup).toMatchSnapshot();
  });
});
