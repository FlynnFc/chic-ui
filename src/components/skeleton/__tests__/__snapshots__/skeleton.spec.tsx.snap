// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton Component Renders Correctly 1`] = `
Array [
  <div
    className="sc-bdfBQB dzfvNA"
    height={100}
    width={100}
  />,
  <div
    className="sc-bdfBQB gIIwQU"
    height={60}
    width={80}
  />,
  <div
    className="sc-bdfBQB bDFWYu"
    height={100}
    width={100}
  />,
]
`;
